### PR TITLE
[Transform] Submit initial draft of README 

### PR DESCRIPTION
**Pull request by wooksong**
This PR fills the README.md file for tensor_transform.

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_transform/README.md [hunk 1]
```diff
@@ -1 +1,77 @@
...
+   - other/tensor
+
+## Properties
+
+- mode (readable, writable): Mode used for transforming tensor
+  - Enum "gtt_mode_type" Default: -1, "unknown"
+    - (0): dimchg
+      - A mode for changing tensor dimensions
+      - An option should be provided as option=FROM_DIM:TO_DIM (with a regex, ^([0-3]):([0-3])$, where NNS_TENSOR_RANK_LIMIT is 4).
+      - Example: Move 1st dim to 2nd dim (i.e., I.e., [a][H][W][C] ==> [a][C][H][W])
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_transform/README.md]: i.e is duplicated

File: gst/nnstreamer/tensor_transform/README.md [hunk 2]
```diff
@@ -1 +1,77 @@
 # NNStreamer::tensor_transform
+
+## Supported Features
+
+- Transformation the shape or data type of ```other/tensor``` stream.
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_transform/README.md]: ```s/- Transformation the shape or data type of ```other/tensor``` stream./- Transformation the shape, data values (arithmetics or normalization), or data type of ```other/tensor``` stream./```

File: gst/nnstreamer/tensor_transform/README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+- If possible, the tensor_transform element exploits [ORC: Optimized inner Loop Runtime Compiler](https://gitlab.freedesktop.org/gstreamer/orc) to accelerate the supported operations.
+
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_transform/README.md]: - Aggregate multiple operators into a single transform instance for performance optimization.
    - E.g., ``` tensor_transform mode=typecast option=uint8 ! tensor_transform mode=arithmetic option=mul:4 ! tensor_transform mode=arithmetic option=add:25 ``` can be optimized by ``` tensor_transform mode=arithmetic option=typecast:uint8,mul:8,add:25```.

File: gst/nnstreamer/tensor_transform/README.md (hunk 1 above)
Comment (wooksong) [reply gst/nnstreamer/tensor_transform/README.md]: Corrected. Thnx!

File: gst/nnstreamer/tensor_transform/README.md (hunk 3 above)
Comment (wooksong) [reply gst/nnstreamer/tensor_transform/README.md]: Added.

File: gst/nnstreamer/tensor_transform/README.md (hunk 2 above)
Comment (wooksong) [reply gst/nnstreamer/tensor_transform/README.md]: Modified.

File: gst/nnstreamer/tensor_transform/README.md [hunk 4]
```diff
@@ -1 +1,79 @@
...
+- SRC
+
+   - One always source pad named 'src'
+   - other/tensor
+
+## Properties
+
+- mode (readable, writable): Mode used for transforming tensor
+  - Enum "gtt_mode_type" Default: -1, "unknown"
```
Comment (leemgs) [inline gst/nnstreamer/tensor_transform/README.md]: > http://nnsuite.mooo.com/nnstreamer/ci/repo-workers/pr-checker/2111-202002181711090.57120490074158-9f1a68178074c31d11683a4a6e1f46281d429148/report/misspelling_result.txt

Looks good, BTW, what is meaning of  the "**gtt**" of "gtt_mode_type"?

Comment (wooksong) [reply gst/nnstreamer/tensor_transform/README.md]: @leemgs GstTensorTransform?

**Pull request merged**